src/components/portal/BottomSheet.tsx

```diff
@@ -0,0 +1,75 @@
+import { type ComponentProps, type MouseEventHandler } from 'react';
+import { css } from '@emotion/react';
+import { m, type Variants } from 'framer-motion';
+
+import { defaultEasing, defaultFadeInVariants } from '~/constants/motions';
+
+import AnimatePortal from './AnimatePortal';
+import { scrimCss } from './scrim.style';
+
+interface Props extends ComponentProps<typeof AnimatePortal> {
+  onClickScrim?: VoidFunction;
+}
+
+const BottomSheet = ({ onClickScrim, isShowing, children, mode }: Props) => {
+  const onClickScrimDefault: MouseEventHandler<HTMLDivElement> = (e) => {
+    if (e.target !== e.currentTarget) return;
+    if (onClickScrim) onClickScrim();
+  };
```

Comment:
```suggestion
const onClickScrimDefault: MouseEventHandler<HTMLDivElement> = (e) => {
  if (e.target === e.currentTarget && onClickScrim) {
    onClickScrim();
  }
};
```

```suggestion
const onClickScrimDefault: MouseEventHandler<HTMLDivElement> = (e) => e.target === e.currentTarget && onClickScrim && onClickScrim();
```

굳이 짧게 하려면 이렇게도 쓸 수 있을 것 같은데... 저는 지금 코드가 훨씬 보기 좋은 것 같아요!